manifests/init.pp

```diff
@@ -35,8 +35,8 @@
 
   String                      $group         = 'dehydrated',
   String                      $user          = 'dehydrated',
-  String                      $repo_source   = 'https://github.com/lukas2511/dehydrated.git',
-  String                      $repo_revision = 'v0.5.0',
+  String                      $repo_source   = 'https://github.com/dehydrated-io/dehydrated.git',
```

Comment:
@opus-codium/core changing the remote url makes vcsrepo unhappy. I think we should revert this part of the change, release a new minor version, than update the repo url and release a new major version.

Reply:
OK